### PR TITLE
fix logging jobs to use envvars

### PR DESCRIPTION
**Pull request by jcantrill**
@stevekuznetsov I think this would superceded #1181. It utilizes the pattern of reusing the jobs as you described with overriding the repos. It removes the parts I think @richm added to disable repos but that I believe is moot given the changes identified by @ewolinetz in #1181.

Please have a look.

### CODE REVIEW

**Comment by jcantrill** [comment]
reopening #1186

File: sjb/config/test_cases/test_branch_openshift_ansible_logging.yml [hunk 1]
```diff
@@ -192,11 +31,7 @@ extensions:
         popd
         logging_extras=""
         if [[ "${PULL_REFS%%:*}" == es5.x ]]; then
```
Comment (stevekuznetsov) [inline sjb/config/test_cases/test_branch_openshift_ansible_logging.yml]: you could also add `$EXTRA_EVARS` to your `ansible-playbook` call and pass these in only in the es5.x job, your call

File: sjb/config/test_cases/test_branch_openshift_ansible_logging_json_file.yml [hunk 2]
```diff
@@ -1,5 +1,11 @@
 ---
-parent: 'common/test_cases/origin_built_release.yml'
+parent: 'common/test_cases/origin_built_installed_release.yml'
+overrides:
+  sync_repos:
```
Comment (stevekuznetsov) [inline sjb/config/test_cases/test_branch_openshift_ansible_logging_json_file.yml]: i would expect this to be the same as in the other one -- an override on `sync:`

File: sjb/config/test_cases/test_pull_request_openshift_ansible_logging_37.yml [hunk 3]
```diff
@@ -5,3 +5,4 @@ overrides:
     - "openshift,origin=release-3.7"
     - "openshift,aos-cd-jobs=master"
     - "openshift,origin-aggregated-logging=release-3.7"
+  evars: "-e openshift_disable_check=* -e openshift_install_examples=false"
```
Comment (stevekuznetsov) [inline sjb/config/test_cases/test_pull_request_openshift_ansible_logging_37.yml]: if `test_cases/test_branch_openshift_ansible_logging.yml` exposes these, I don't think you will need them in each child?

File: sjb/config/test_cases/test_pull_request_origin_aggregated_logging_journald_37.yml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+---
```
Comment (stevekuznetsov) [inline sjb/config/test_cases/test_pull_request_origin_aggregated_logging_journald_37.yml]: If these are re-names, maybe we can do that in a separate PR? this one is big enough

File: sjb/config/test_cases/test_branch_openshift_ansible_logging.yml (hunk 1 above)
Comment (richm) [reply sjb/config/test_cases/test_branch_openshift_ansible_logging.yml]: We really want to merge es5.x into master (for 3.10) ASAP

File: sjb/config/test_cases/test_pull_request_origin_aggregated_logging_journald.yml [hunk 5]
```diff
@@ -0,0 +1,71 @@
+---
+parent: 'common/test_cases/origin_built_installed_release.yml'
+overrides:
+  sync:
+    - "openshift,aos-cd-jobs=master"
+    - "openshift,openshift-ansible=master"
+  evars: "-e skip_sanity_checks=true"
```
Comment (richm) [inline sjb/config/test_cases/test_pull_request_origin_aggregated_logging_journald.yml]: need to tell origin install to use journald - add `-e openshift_docker_log_driver=journald -e openshift_docker_options=--log-driver=journald`
likewise for the json-file jobs - add `-e openshift_docker_log_driver=json-file -e openshift_docker_options=--log-driver=json-file`

File: sjb/config/test_cases/test_branch_openshift_ansible_logging_json_file.yml (hunk 2 above)
Comment (stevekuznetsov) [reply sjb/config/test_cases/test_branch_openshift_ansible_logging_json_file.yml]: outstanding

File: sjb/config/test_cases/test_pull_request_openshift_ansible_logging.yml [hunk 6]
```diff
@@ -1,3 +1,4 @@
 ---
 parent: 'test_cases/test_branch_openshift_ansible_logging.yml'
-overrides: {}
\ No newline at end of file
+overrides:
+  evars: "-e skip_sanity_checks=true"
```
Comment (stevekuznetsov) [inline sjb/config/test_cases/test_pull_request_openshift_ansible_logging.yml]: So the parent has ` +  evars: "-e skip_sanity_checks=true -e openshift_disable_check=* -e openshift_install_examples=false" ` so this child will remove some of those? why?

**Comment by jcantrill** [comment]
close in favor of #1220
